src/i_video.c

```diff
@@ -1616,6 +1615,10 @@ static void I_InitGraphicsMode(void)
             flags |= SDL_WINDOW_FULLSCREEN_DESKTOP;
         }
     }
+    else // !fullscreen
+    {
+        flags |= SDL_WINDOW_RESIZABLE;
```

Comment:
Does `SDL_RenderSetLogicalSize()` work at all if this flag is not set?

Reply:
Yes, seems to work fine.